CMakeLists.txt

```diff
@@ -497,12 +497,11 @@ if (BUILD_TOOLS)
     ${WABT_SOURCE_DIR}/src/template/wasm2c.declarations.c
     )
 
-  add_library(cwriter-template wasm2c_header_top.cc wasm2c_header_bottom.cc wasm2c_source_includes.cc wasm2c_source_declarations.cc)
+    set(CWRITER_TEMPLATE_SRC wasm2c_header_top.cc wasm2c_header_bottom.cc wasm2c_source_includes.cc wasm2c_source_declarations.cc)
```

Comment:
Looks like there is some extra space here at the beginning of the line...

Reply:
> lgtm. It would be nice if we had testing to catch this kind of problem before release. :-(
> 
> Longer term, I think we would like to include c-writer.cc and the template sources in libwabt itself (rather than only linking them in to the wasm2c executable), so that programs that link against libwabt can use the CWriter class. But that can wait.

I actually don't know how this happened because I could repro it locally (locally I don't see any `.so` libraries).  I guess maybe there is some kind of cmake flag that says "build any libraries that are not specifically static, as dynamic".. but i odn't know what it is.